include/sys/_pmap.h

```diff
@@ -70,6 +70,15 @@ paddr_t pde_alloc(pmap_t *pmap);
 pde_t pde_make(int lvl, paddr_t pa);
 static inline pde_t *pde_ptr(paddr_t pd, int lvl, vaddr_t va);
 
+static __no_profile inline pde_t *pde_ptr_idx(paddr_t pd_pa, size_t index) {
```

Comment:
If `pde_ptr_idx` prototype has been defined please remove it.

Reply:
I am not sure what you mean by that. Should I create a prototype for that function (because this function doesn't have one)? Or should I remove the prototype for `pde_ptr` and leave only the definitions?

Reply:
Nvm. I had an impression there was a prototype for that function in another header file.